app/views/recipes/index.html.erb

```diff
@@ -1,30 +1,30 @@
 <section class='recipe'>
-  <h1>Recipes List</h1>
-  <div class="container">
-    <button class="button"><%= link_to "Add Recipe", new_recipe_path%></button>
-  </div>
-  <% if @recipes.count > 0%>
-    <table>
-      <tr>
-        <th>Name</th>
-        <th>Preparation_time</th>
-        <th>Cooking_time</th>
-        <th>Description</th>
-        <th>Public</th>
-      </tr>
-      <% @recipes.each do |recipe| %>
-      <tr>
-        <td><%= recipe.name %></td>
-        <td><%= recipe.preparation_time %></td>
-        <td><%= recipe.cooking_time %></td>
-        <td><%= recipe.description %></td>
-        <td><%= recipe.public %></td>
-        <td><%= link_to "Delete", recipe_path(recipe), class: "delete",data: { turbo_method: :delete, turbo_confirm: "Are you  sure?"}%>
-        </td>
-      </tr>
-      <% end %>
-    <%else%>
-    <h2>No recipe</h2>
-  <%end%>
-  </div>
+   <h1>Recipes List</h1>
+   <div class="container">
+      <button class="button"><%= link_to "Add Recipe", new_recipe_path%></button>
+   </div>
+   <% if @recipes.count > 0%>
+      <table>
+         <tr>
+         <th>Name</th>
+         <th>Preparation_time</th>
+         <th>Cooking_time</th>
+         <th>Description</th>
+         <th>Public</th>
+         </tr>
+         <% @recipes.each do |recipe| %>
+         <tr>
+         <td><%= recipe.name %></td>
+         <td><%= recipe.preparation_time %></td>
+         <td><%= recipe.cooking_time %></td>
+         <td><%= recipe.description %></td>
+         <td><%= recipe.public %></td>
+         <td><%= link_to "Delete", recipe_path(recipe), class: "delete",data: { turbo_method: :delete, turbo_confirm: "Are you  sure?"}%>
+         </td>
+         </tr>
```

Comment:
- [ ] [OPTIONAL] Please consider adding a link here so that the recipes are clickable and lead to the recipe show page. This will improve UX 😎 
![image](https://user-images.githubusercontent.com/46839548/219828840-faf88bf5-803a-422c-b185-c3fc6f93a625.png)